src/transformers/models/auto/modeling_auto.py

```diff
@@ -551,6 +554,13 @@
     ]
 )
 
+MODEL_FOR_OBJECT_DETECTION_MAPPING = OrderedDict(
```

Comment:
Create an Auto-class as well? Otherwise there is no point builiding the mapping.